client/.editorconfig

```diff
@@ -0,0 +1,34 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
```

Comment:
Yeah I saw it, but didn't follow the link. seems useful and possibly use for yggdrasil to keep consistent spacing etc.